lms/templates/discussion/_underscore_templates.html

```diff
@@ -389,7 +389,7 @@ <h1 class="home-title">${course.display_name_with_default}</h1>
         ${'<% if (cohort_options) { %>'}
         <div class="post-field">
             <label class="field-label">
-                <span class="field-label-text">
+                <span class="field-label-text" aria-describedby="field_help_visible_to">
```

Comment:
hey @clrux, can you move this to the select element below?  It needs to be on a focusable element to work.